app/views/unattended/report_templates/host_-_last_checkin.erb

```diff
@@ -31,10 +35,6 @@ template_inputs:
   default: 'true'
   hidden_value: false
 model: ReportTemplate
-description: |
-  This report lists hosts that haven't checked in in the specified interval (by default
-  5 hours). By default it also includes hosts we don't have last check information for.
-  The check in is based on the subscription manager communication.
 require:
 - plugin: katello
 ￼ version: 3.16.0
```

Comment:
Since we're at it, could we remove these weird broken symbols? I've noticed it's across a lot of templates we have.